### PR TITLE
add images, improve doc

### PR DESCRIPTION
**Pull request by vvmruder**
I added the images and some doc as requested

### CODE REVIEW

File: doc/source/index.rst [hunk 1]
```diff
@@ -40,6 +40,26 @@ customization.
    :scale: 20 %
    :align: center
 
+The server basically provides access to the 4 services:
```
Comment (jwkaltz) [inline doc/source/index.rst]: The server provides access to the 4 services:

File: doc/source/index.rst [hunk 2]
```diff
@@ -40,6 +40,26 @@ customization.
...
+
+* GetExtractById
+* GetEGRID
+* GetCapabilities
+* GetVersions
+
+The server is able to use ÖEREBlex, GeoAdmin API for gathering data. This need to be configured and prepared
```
Comment (jwkaltz) [inline doc/source/index.rst]: "This needs to be configured"

File: doc/source/index.rst [hunk 3]
```diff
@@ -40,6 +40,26 @@ customization.
...
+by the integrator of this package. In addition a small tool for loading legend symbols for each used WMS layer
+is included.
+
+For the moment we can provide a adapter to get the PDF static extract from mapfishprint.
```
Comment (jwkaltz) [inline doc/source/index.rst]: "For the moment we provide an adapter"

**Comment by vvmruder** [comment]
Everyone is OK with merging this?

**Pull request merged**